benchmark/benchmark.js

```diff
@@ -19,7 +19,7 @@ function tokenize (grammar, content, lineCount) {
     tokenCount += tokenizedLine.length
   }
   const tokensPerMillisecond = Math.round(tokenCount / duration)
-  console.log(`Generated #{tokenCount} tokens for ${lineCount} lines in ${duration}ms (${tokensPerMillisecond} tokens/ms)`)
+  console.log(`Generated ${tokenCount} tokens for ${lineCount} lines in ${duration}ms (${tokensPerMillisecond} tokens/ms)`)
```

Comment:
You might find [this command](https://github.com/Alhadis/Atom-PhoenixTheme/blob/master/src/commands/editor.js#L46-L65) to be helpful. =) It'll automatically prepend a `$` when you insert curly brackets in a JS string literal.